content/docs/state-and-lifecycle.md

```diff
@@ -103,30 +104,30 @@ class Clock extends React.Component {
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/zKRGpo?editors=0010)
 
-`Clock` is now defined as a class rather than a function.
+`Clock` को अब एक फ़ंक्शन के बजाय एक क्लास के रूप में परिभाषित किया गया है।
 
-The `render` method will be called each time an update happens, but as long as we render `<Clock />` into the same DOM node, only a single instance of the `Clock` class will be used. This lets us use additional features such as local state and lifecycle methods.
+हर अपडेट पर `render` फंक्शन कॉल होगा, लेकिन जब तक हम `<Clock />` को एक ही DOM नोड में रेंडर करते हैं, तब तक `Clock` क्लास का केवल एक ही उदाहरण इस्तेमाल किया जाएगा। इससे हम लोकल state और Lifecycle विधियों जैसी अतिरिक्त सुविधाओं का उपयोग कर सकते हैं।
 
-## Adding Local State to a Class {#adding-local-state-to-a-class}
+## क्लास में लोकल state ऐड करना {#adding-local-state-to-a-class}
```

Comment:
instead of add karna, it could be daalna?

Reply:
"ऐड करना" is giving it a more descriptive title but I can change it if you want.

Reply:
we can let it be then. My thought was it was a mixture of hindi and english this way